cardano-node/ChangeLog.md

```diff
@@ -1,5 +1,24 @@
 # Changelog for cardano-node
 
+## 1.21.1 -- September 2020
+
+### node changes
+- Fix configuration defaults that was interfering with using systemd socket
+  activation (#1927)
+- Workaround for building on Linux systems without systemd (#1775)
+- Fix the severity level for some protocol tracers (#1910)
```

Comment:
@jbgi can you confirm logs look fine after this change?

Reply:
looks good, no apparent change in logs.